students/JonSerpas/lesson03/assignment/src/basic_operations.py

```diff
@@ -0,0 +1,122 @@
+import peewee
```

Comment:
Be careful with catching general Exceptions. Similar to the previous assignment, you want to catch specific exceptions so you know your program is behaving as expected and you specifically handle those error cases